digitalstrategie/settings/base.py

```diff
@@ -111,9 +112,22 @@
 
 LANGUAGES = [
     ('en', _('English')),
-    ('de', _('German'))
+    ('de', _('German')),
+    ('de-ls', _('Easy German'))
 ]
 
+# adding language info for easy german
+EXTRA_LANG_INFO = {
+    'de-ls': {
+        'bidi': False,
+        'code': 'de-ls',
+        'name': 'Easy German',
+        'name_local': u"Leichte Sprache",
```

Comment:
do we really need the `u` ?

Reply:
I found somewhere that it needs to be unicode and in Aplus we also did it like that.. but to be honest not sure if that is needed, it also works without.. should I remove it you think?

Reply:
I don't really know about it in a Django context, but afaik it was required in python2 - in python3 this become unnecessary, so yes, if it works I think we should remove it

Reply:
Done!